modules/op-installing-pipelines-as-code-on-an-openshift-cluster.adoc

```diff
@@ -7,70 +7,27 @@
 = Installing {pac} on an {product-title}
 
 [role="_abstract"]
-{pac} is installed by default when you install the {pipelines-title} Operator. If you are using {pipelines-shortname} 1.7 or later versions, skip the procedure for manual installation of {pac}.
+{pac} is installed in the `openshift-pipelines` namespace when you install the {pipelines-title} Operator. For more details, see _Installing OpenShift Pipelines_ in the _Additional resources_ section.
 
-However, if you want to disable the default installation of {pac} with the {pipelines-title} Operator, set the value of the `enablePipelinesAsCode` field as `false` in the `TektonConfig` custom resource.
+To enable the default installation of {pac} with the {pipelines-title} Operator, the value of the `enable` parameter is set to `true` in the `TektonConfig` custom resource:
 
 [source,yaml]
 ----
 ...
-spec:
-    addon:
-      enablePipelinesAsCode: false
+ spec:
+    platforms:
+      openshift:
+        pipelinesAsCode:
+          enable: true
+          settings:
+            application-name: SME
```

Comment:
Default config in the latest build of Openshift Pipelines 1.9.0 shows `application-name: Pipelines as Code CI`

Reply:
^^ @sm43